write.go

```diff
@@ -188,6 +188,7 @@ func writeElement(w dicomio.Writer, elem *Element, opts writeOptSet) error {
 		return err
 	}
 
+	w.GetTransferSyntax()
```

Comment:
can you tell me more about why this is needed?

Reply:
Whoops, having some logic there, then removed it in a refactor and missed that line, thanks for the catch!